<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TGEventsConfigurationWidget</class>
 <widget class="QWidget" name="TGEventsConfigurationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="swEventsData">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pgScheduleEvent">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_12">
         <property name="title">
          <string>ID события</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QLineEdit" name="leScheduledEventID"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_21">
         <property name="title">
          <string>Условия</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_36">
          <item>
           <widget class="QFrame" name="frame_20">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_16">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>6</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>6</number>
             </property>
             <item>
              <widget class="QFrame" name="frame_22">
               <property name="frameShape">
                <enum>QFrame::Shape::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Shadow::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_38">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>10</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Периодичность</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="cbScheduledEventType">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>25</height>
                   </size>
                  </property>
                  <item>
                   <property name="text">
                    <string>Ежечасно</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Ежедневно</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Еженедельно</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Ежемесячно</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Ежегодно</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Orientation::Vertical</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Policy::Maximum</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_23">
               <property name="frameShape">
                <enum>QFrame::Shape::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Shadow::Raised</enum>
               </property>
               <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,0,0,0" columnminimumwidth="0,0,0,0">
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item row="2" column="3">
                 <widget class="QComboBox" name="cbScheduledEventDoW">
                  <item>
                   <property name="text">
                    <string>Понедельник</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Вторник</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Среда</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Четверг</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Пятница</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Суббота</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Воскресенье</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="lbScheduledEventMonth">
                  <property name="text">
                   <string>Месяц</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="lbScheduledEventMinute">
                  <property name="text">
                   <string>Минута</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3">
                 <widget class="QComboBox" name="cbScheduledEventMonth">
                  <item>
                   <property name="text">
                    <string>Январь</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Февраль</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Март</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Апрель</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Май</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Июнь</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Июль</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Август</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Сентябрь</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Октябрь</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Ноябрь</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Декабрь</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="3" column="3">
                 <widget class="QSpinBox" name="sbScheduledEventHour">
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignmentFlag::AlignCenter</set>
                  </property>
                  <property name="maximum">
                   <number>23</number>
                  </property>
                 </widget>
                </item>
                <item row="1" column="3">
                 <widget class="QSpinBox" name="sbScheduledEventDay">
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignmentFlag::AlignCenter</set>
                  </property>
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>31</number>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="lbScheduledEventDay">
                  <property name="text">
                   <string>День</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0" colspan="3">
                 <widget class="QLabel" name="lbScheduledEventDoW">
                  <property name="text">
                   <string>День недели</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="lbScheduledEventHour">
                  <property name="text">
                   <string>Час</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="3">
                 <widget class="QSpinBox" name="sbScheduledEventMinute">
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignmentFlag::AlignCenter</set>
                  </property>
                  <property name="maximum">
                   <number>59</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_21">
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_17" stretch="0,0">
             <property name="spacing">
              <number>20</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>6</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>6</number>
             </property>
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Минимальный уровень авторизации</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cbTGScheduledEventConfUserLevel">
               <property name="currentIndex">
                <number>0</number>
               </property>
               <item>
                <property name="text">
                 <string>Все пользователи</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Зарегистрированные пользователи</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Каналы</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Администратор</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbScheduledEventsActions">
         <property name="title">
          <string>Действия</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_37"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pgTriggerEvent">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_9">
         <property name="title">
          <string>ID события</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLineEdit" name="leEventID"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_10">
         <property name="title">
          <string>Переменная</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QTableWidget" name="twOPCTagsEvents"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_11">
         <property name="title">
          <string>Условия</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_20">
          <item>
           <widget class="QFrame" name="frame_15">
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>6</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>6</number>
             </property>
             <property name="horizontalSpacing">
              <number>20</number>
             </property>
             <property name="verticalSpacing">
              <number>10</number>
             </property>
             <item row="1" column="3">
              <widget class="QLineEdit" name="leTagCompareValueHysterezis">
               <property name="maxLength">
                <number>8</number>
               </property>
               <property name="cursorPosition">
                <number>0</number>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLineEdit" name="leTagNameEvent">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Тип сравнения</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="cbTGEventConfCompareType">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="currentIndex">
                <number>0</number>
               </property>
               <item>
                <property name="text">
                 <string>Равно ==</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Не равно &lt;&gt;</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Больше &gt;</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Меньше &lt;</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Изменился</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Значение</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Гистерезис</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLineEdit" name="leTagCompareValue">
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Переменная</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_16">
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="0,0">
             <property name="spacing">
              <number>20</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>6</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>6</number>
             </property>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Минимальный уровень авторизации</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cbTGEventConfUserLevel">
               <property name="currentIndex">
                <number>0</number>
               </property>
               <item>
                <property name="text">
                 <string>Все пользователи</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Зарегистрированные пользователи</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Каналы</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Администратор</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbEventsActions">
         <property name="title">
          <string>Действия</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_18"/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
